src/modules/emailMethods.ts

```diff
@@ -0,0 +1,194 @@
+                data.hire ? `hire per ${data.grade} level/grade` : 'do not hire'
+            } candidate ${section.interview.candidate.name}\n
+            ${data.feedback}\n
+            ${config.defaultPageURL}/${generatePath(Paths.SECTION, {
+                interviewId: section.interviewId,
+                sectionId: section.id,
+            })}`,
+        });
+    }
+};
+
+export const cancelSectionEmail = async (sectionId: number) => {
+    const section = await prisma.section.findFirstOrThrow({
+        where: { id: sectionId },
+        include: {
+            interview: { include: { candidate: true } },
+            calendarSlot: true,
+            interviewer: true,
+            sectionType: true,
+        },
+    });
+
+    const date = section.calendarSlot?.date ? formatDateToLocaleString(section.calendarSlot?.date) : '';
+
+    let cancelIcalEvent;
+    if (section.calendarSlotId) {
+        const { originalDate, ...restException } = await calendarMethods.getEventExceptionById(section.calendarSlotId);
+        const icalEventDataCancelException = createIcalEventData({
+            id: section.calendarSlotId,
+            users: [userOfEvent(section.interviewer, null)],
+            start: restException.date,
+            description: restException.eventDetails.description,
+            duration: restException.eventDetails.duration,
+            summary: restException.eventDetails.title,
+        });
+
+        cancelIcalEvent = calendarEvents({
+            method: ICalCalendarMethod.CANCEL,
+            events: [icalEventDataCancelException],
+        });
+
+        const { eventDetails, ...series } = await calendarMethods.getEventById(restException.eventId);
+
+        const exclude = [
+            ...series.exceptions
+                .map((exception) => exception.originalDate)
+                .filter((date) => date.toTimeString() !== originalDate.toTimeString()),
+            ...series.cancellations.map((cancelletion) => cancelletion.originalDate),
+        ];
+
+        const rRule = RRule.fromString(series.rule);
+
+        const icalEventDataUpdateEvent = createIcalEventData({
+            id: series.id,
+            users: [userOfEvent(section.interviewer, null)],
+            start: rRule.options.dtstart,
+            duration: eventDetails.duration,
+            description: eventDetails.description,
+            summary: eventDetails.title,
+            rule: rRule.options.freq,
+            exclude: exclude.length ? exclude : undefined,
+        });
+
+        await sendMail({
+            from: 'Hire',
+            to: section.interviewer.email,
+            subject: eventDetails.title,
+            text: '',
+            icalEvent: calendarEvents({
+                method: ICalCalendarMethod.REQUEST,
+                events: [icalEventDataUpdateEvent],
+            }),
+        });
+    }
+
+    return sendMail({
+        from: 'Taskany Hire',
```

Comment:
you have copypaste 'from' and other fields. Make a wrapper function with default values.